Sources/Clappr_iOS/Classes/Animation/DoubleTapAnimation.swift

```diff
@@ -0,0 +1,254 @@
+import UIKit
+
+class DoubleTapAnimation {
+    private var backLabel = UILabel()
+    private var fowardLabel = UILabel()
+    private var core: Core?
+    
+    private let playImage = UIImage.fromName("play", for: PlayButton.self)
+    
+    private lazy var backIcon1 = mirrorImage(UIImageView(image: playImage))
+    private lazy var backIcon2 = mirrorImage(UIImageView(image: playImage))
+    private lazy var backIcon3 = mirrorImage(UIImageView(image: playImage))
+    
+    private lazy var fowardIcon1 = UIImageView(image: playImage)
+    private lazy var fowardIcon2 = UIImageView(image: playImage)
+    private lazy var fowardIcon3 = UIImageView(image: playImage)
+    
+    private var leftBubbleView = UIView()
+    private var rightBubbleView = UIView()
+    
+    private var leftBubbleHeight = NSLayoutConstraint()
+    private var leftBubbleWidth = NSLayoutConstraint()
+    
+    private var rightBubbleHeight = NSLayoutConstraint()
+    private var rightBubbleWidth = NSLayoutConstraint()
+    
+    init(_ core: Core?) {
+        self.core = core
```

Comment:
Wouldn't be better if this class just represented the view itself?

A view that contained the left bubble and the right bubble?

The logic of adding the view, bringing to the front and so on would be made anywhere else. Here we coupled this class to the Core and if in the future we want to move it to anywhere else we would have to change it.

Making it not-specialized will give us the flexibility of use it anywhere we want. What do you think?